includes/classes/Rest/IssuesAPI.php

```
@@ -0,0 +1,479 @@
+				return new \WP_Error( 'rest_issue_invalid_fields', __( 'Missing required fields.', 'accessibility-checker' ), [ 'status' => 400 ] );
+			}
+		}
+
+		$post = get_post( $request['postid'] );
+		if ( ! $post || ! is_a( $post, '\WP_Post' ) ) {
+			return new \WP_Error( 'rest_issue_invalid_post', __( 'Invalid post ID.', 'accessibility-checker' ), [ 'status' => 400 ] );
+		}
+
+		// Create a new issue.
+		// NOTE: the return values of this is strange, need to find a better way to validate what happened.
+		$inserted = ( new Insert_Rule_Data() )->insert( $post, $request['rule'], $request['ruletype'], $request['object'], $request['user'] );
```

PHPStan (2.0.3)

Method EDAC\Admin\Insert_Rule_Data::insert() invoked with 5 parameters, 4 required.


(arguments.count)